### PR TITLE
10-7959f-1 Personal Info Page

### PR DESCRIPTION
**Pull request by reiting**
## Summary
This PR created the personal info page for the FMP. The page has a (required) First and Last Name fields, a middle initial field, and a (required) DOB section. 

## Related issue(s)
- https://github.com/department-of-veterans-affairs/va.gov-team/issues/76792
- 
## Testing done

- Completed running tests to ensure that all passed
- Manual tests
- 
## Screenshots
Screenshot 2024-02-27 at 10 44 34 AM

## What areas of the site does it impact?
ivc-champva forms section

### Quality Assurance & Testing

- [ ] I fixed|updated|added unit tests and integration tests for each feature (if applicable).
- [x] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs
- [x] Linting warnings have been addressed
- [ ] Documentation has been updated ([link to documentation](#) \*if necessary)
- [x] Screenshot of the developed feature is added
- [ ] [Accessibility testing](https://depo-platform-documentation.scrollhelp.site/developer-docs/wcag-2-1-success-criteria-and-foundational-testing) has been performed

### Error Handling

- [x] Browser console contains no warnings or errors.
- [ ] Events are being sent to the appropriate logging solution
- [ ] Feature/bug has a monitor built into Datadog or Grafana (if applicable)

### Authentication

- [ ] Did you login to a local build and verify all authenticated routes work as expected with a test user

### :warning: Team Sites (only applies to modifications made to the VA.gov header) :warning:

- [ ] The vets-website header does not contain any web-components
- [ ] I used the [proxy-rewrite steps](https://github.com/department-of-veterans-affairs/vets-website/tree/main/src/applications/proxy-rewrite#local-dev) to test the injected header scenario
- [ ] I reached out in the `#sitewide-public-websites` Slack channel for questions

## Requested Feedback

(OPTIONAL) _What should the reviewers know in addition to the above. Is there anything specific you wish the reviewer to assist with. Do you have any concerns with this PR, why?_

### CODE REVIEW

**Review by balexandr — DISMISSED** [review]
easy lgtm

**Review by michaelclement — DISMISSED** [review]
lgtm!

**Review by cloudmagic80 — DISMISSED** [review]
Looks good to me

File: src/applications/ivc-champva/10-7959f-1/config/form.js [hunk 1]
```diff
@@ -60,21 +63,26 @@ const formConfig = {
...
-            fullNameTitle: inlineTitleUI('Your name'),
+            ...titleUI(
+              `Veteran's personal information`,
+              `We use this information to contact you and verify other details.`,
+            ),
             fullName: fullNameUI(),
+            veteranDOB: dateOfBirthUI(),
           },
           schema: {
             type: 'object',
```
Comment (micahchiang) [inline src/applications/ivc-champva/10-7959f-1/config/form.js:80]: It's a nit, but can these back ticks be changed to quotes?

Comment (reiting) [reply src/applications/ivc-champva/10-7959f-1/config/form.js]: Okay. Are back ticks not allowed (just for my future personal knowledge)

**Review by michaelclement — APPROVED** [review]

**Review by balexandr — APPROVED** [review]
lgtm

**Review by cloudmagic80 — APPROVED** [review]
Looks good

**Review by Andrew565 — APPROVED** [review]

**Pull request merged**